server-manager.proto

```diff
@@ -56,24 +55,22 @@ package CartesiServerManager;
 // to the machine: i.e., from the perspective of the machine, it never happened.
 // State inspection is synchronous: the query is processed as soon as the current input is done (if any) and all
 // produced Reports are immediately returned.
-// A Cartesi Machine used with the Cartesi Server Manager must define 5 memory ranges, in addition to however
+// A Cartesi Machine used with the Cartesi Server Manager must define 2 memory ranges, in addition to however
 // many flash drives the application it runs might require.
 // Each memory range must have a power-of-two length and must start at a multiple of its size.
-// The first two are the rx buffer and tx buffer memory ranges.
-// The rx buffer is used by the Cartesi Server Manager to send the input and query payloads into the Cartesi
-// Machine.
-// The tx buffer is used by the Cartesi Machine to send Vouchers, Notices, and Reports to the Cartesi Server
-// Manager.
-// Each input to AdvanceState, in addition to the payload, requires some metadata.
-// This is sent into the Cartesi Machine through the input metadata memory range.
-// The last two remaining memory ranges are the voucher hashes and notice hashes memory ranges.
-// These contain an array of hashes, respectively of each voucher and notice produced while processing a given input.
-// Whenever an input is processed, the Cartesi Server Manager collects from the Cartesi Machine a Merkle proof
-// that the voucher hashes and notice hashes memory ranges are part of the Cartesi Machine State.
-// Likewise, each voucher and notice accompanies a proof that its hash is part of, respectively, the voucher hashes
-// memory range and notice hashes memory range.
-// Finally, the Cartesi Server Manager also maintains two additional Merkle trees, respectively containing as
-// leaves the Merkle tree root hash of the voucher hashes memory range and the notice hashes memory range.
+// These are  rx buffer and tx buffer memory ranges.
+// The rx buffer is used by the Cartesi Server Manager to send the input and query data into the Cartesi Machine.
+// The tx buffer is used by the Cartesi Machine to send Vouchers, Notices, and Reports to the Cartesi Server Manager.
+// Each input data to AdvanceState includes the old payload and metadata, combined.
+// As each input is processed, it may generate many outputs (voucher or notice) that are collected by the
```

Comment:
To me, it sounds weird to exemplify types of outputs in singular form.

```suggestion
// As each input is processed, it may generate many outputs (which can be vouchers or notices) that are collected by the
```